yarn.lock

```diff
@@ -4,37 +4,35 @@
 
 "@types/node@^10.12.0":
   version "10.17.60"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.17.60.tgz#35f3d6213daed95da7f0f73e75bcc6980e90597b"
+  resolved "https://registry.npmjs.org/@types/node/-/node-10.17.60.tgz"
```

Comment:
Hmm why did these change? Any ideas?

Reply:
It seems like I'm running an old version of yarn (1.22.19) and I'm guessing you're running a newer one